core/lib/dal/src/witness_generator_dal.rs

```diff
@@ -844,57 +810,6 @@ impl WitnessGeneratorDal<'_, '_> {
         }
     }
 
-    pub async fn mark_leaf_aggregation_gcs_blobs_as_cleaned(&mut self, l1_batch_numbers: Vec<i64>) {
-        {
-            sqlx::query!(
-                r#"
-                UPDATE leaf_aggregation_witness_jobs
-                SET is_blob_cleaned=TRUE
```

Comment:
A few questions that are probably out of the scope for this PR.
1. Do we need a migration where we drop this column?
2. Also, do we want to keep all the records in this table forever, or maybe we want to drop old entries after some time?